mbtb_app/web_portal/mbtb_portal/test/test/README.md

```diff
@@ -10,6 +10,11 @@ using [protractor](https://www.protractortest.org/#/) for angular testing
 npm install -g protractor
 ```
```

Comment:
6 -> Is there any step related to browser set-up before we perform all these steps? (e.g. we bind this webdriver with safari or brave browser)